currency_monthly_rate/README.rst

```diff
@@ -0,0 +1,79 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+=====================
+Currency Monthly Rate
+=====================
+
+This module extends the functionality of base module to support monthly average
+of currency rates.
+
+Installation
+============
+
+Just install it.
```

Comment:
![image](https://user-images.githubusercontent.com/34241889/36848597-630c272a-1d6a-11e8-89f2-43ddf508d3e2.png)
is that section of any use? :smile: